rust-wasm-id-allocator/wasm-id-allocator/src/lib.rs

```diff
@@ -205,3 +202,219 @@ impl InteropIdRange {
         self.count
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use distributed_id_allocator::compressor::{NormalizationError, SessionTokenError};
+
+    const _STABLE_IDS: &[&str] = &[
```

Comment:
Am I wrong in thinking we only use the first and fourth element of this array? Other than that, PR looks good :)